tensorflow/lite/micro/examples/micro_speech/zephyr_riscv/CMakeLists.txt

```diff
@@ -17,50 +27,123 @@ list(REMOVE_ITEM MICRO_SOURCES ${MICRO_TESTS_SOURCES})
+          ${MICRO_SPEECH}/main_functions.cc
+          ${MICRO_SPEECH}/main_functions.h
+          ${MICRO_SPEECH}/recognize_commands.cc
+          ${MICRO_SPEECH}/recognize_commands.h
+          ${MICRO_SPEECH}/no_1000ms_sample_data.cc
+          ${MICRO_SPEECH}/no_1000ms_sample_data.h
+          ${MICRO_SPEECH}/yes_1000ms_sample_data.cc
+          ${MICRO_SPEECH}/yes_1000ms_sample_data.h
+          # ${MICRO_SPEECH}/no_30ms_sample_data.cc
+          # ${MICRO_SPEECH}/no_30ms_sample_data.h
+          # ${MICRO_SPEECH}/yes_30ms_sample_data.cc
+		  # ${MICRO_SPEECH}/yes_30ms_sample_data.h
+		  
+          # micro_features
+          ${MICRO_FEATURES}/tiny_conv_micro_features_model_data.cc
+          ${MICRO_FEATURES}/tiny_conv_micro_features_model_data.h
+          ${MICRO_FEATURES}/micro_model_settings.cc
+          ${MICRO_FEATURES}/micro_model_settings.h
+          ${MICRO_FEATURES}/micro_features_generator.cc
+          ${MICRO_FEATURES}/micro_features_generator.h
+          # ${MICRO_FEATURES}/yes_micro_features_data.cc
+          # ${MICRO_FEATURES}/yes_micro_features_data.h
+          # ${MICRO_FEATURES}/no_micro_features_data.cc
+          # ${MICRO_FEATURES}/no_micro_features_data.h
+          # ${MICRO_FEATURES}/no_feature_data_slice.cc
+          # ${MICRO_FEATURES}/no_feature_data_slice.h
+          # ${MICRO_FEATURES}/yes_feature_data_slice.cc
+		  # ${MICRO_FEATURES}/yes_feature_data_slice.h
+		  
+          # kiss_fft
+          ${KISS_FFT}/kiss_fft.c
+          ${KISS_FFT}/kiss_fft.h
+          ${KISS_FFT}/_kiss_fft_guts.h
+          ${KISS_FFT}/tools/kiss_fftr.c
+		  ${KISS_FFT}/tools/kiss_fftr.h
+		  
+          # microfrontend
+          ${MICROFRONTEND}/fft.cc
+          ${MICROFRONTEND}/fft_util.cc
+          ${MICROFRONTEND}/filterbank.c
+          ${MICROFRONTEND}/filterbank_util.c
+          ${MICROFRONTEND}/frontend.c
+          ${MICROFRONTEND}/frontend_util.c
+          ${MICROFRONTEND}/log_lut.c
+          ${MICROFRONTEND}/log_scale.c
+          ${MICROFRONTEND}/log_scale_util.c
+          ${MICROFRONTEND}/noise_reduction.c
+          ${MICROFRONTEND}/noise_reduction_util.c
+          ${MICROFRONTEND}/pcan_gain_control.c
+          ${MICROFRONTEND}/pcan_gain_control_util.c
+          ${MICROFRONTEND}/window.c
+          ${MICROFRONTEND}/window_util.c
+          ${MICROFRONTEND}/bits.h
+          ${MICROFRONTEND}/fft.h
+          ${MICROFRONTEND}/fft_util
+          ${MICROFRONTEND}/filterbank.h
+          ${MICROFRONTEND}/filterbank_util.h
+          ${MICROFRONTEND}/frontend.h
+          ${MICROFRONTEND}/frontend_util.h
+          ${MICROFRONTEND}/log_lut.h
+          ${MICROFRONTEND}/log_scale.h
+          ${MICROFRONTEND}/log_scale_util.h
+          ${MICROFRONTEND}/noise_reduction.h
+          ${MICROFRONTEND}/noise_reduction_util.h
+          ${MICROFRONTEND}/pcan_gain_control.h
+          ${MICROFRONTEND}/pcan_gain_control_util.h
+          ${MICROFRONTEND}/window.h
+		  ${MICROFRONTEND}/window_util.h
```

Comment:
Indentation ^^

Reply:
Let's talk about it in the first conversation about indentation.